File: Coding/Python/concat_lists.py
# Program to concatenate two lists together
def concatenate_lists ( list1, list2 ):
    all_lists = sum([list1, list2], [])
    return all_lists

list1 = ['a','b','c']
list2 = [1,2,3]
new_list = concatenate_lists ( list1, list2 )
print new_list

File: Coding/Ruby/concat_lists.rb
# Program to concatenate two lists together
def concatenate_lists(list1,list2)
    concat_list = [list1,list2].flatten
    return concat_list
end

list1 = ['a','b','c']
list2 = [1,2,3]
new_list = concatenate_lists(list1,list2)

puts "#{new_list}"